net/keepalived/files/usr/share/keepalived/scripts/rsync.sh

```diff
@@ -0,0 +1,137 @@
+#!/bin/sh
```

Comment:
Can you please run the script through the shellcheck.
I got a few warnings.
We should start right away and not fix it afterwards.

Reply:
Updated